src/helpers/get-newest-month.ts

```diff
@@ -0,0 +1,11 @@
+const getNewestMonth = () => {
+  const lastDataAvailableMonth = new Date();
+  lastDataAvailableMonth.setDate(0);
+  return (
+    lastDataAvailableMonth.getFullYear() +
+    '-' +
+    (lastDataAvailableMonth.getMonth() + 1).toString().padStart(2, '0')
+  );
+};
+
+export default getNewestMonth;
```

Comment:
This way to get the lastDataAvailableMonth is not robust. A solid way is implemented in #635, which you can refer to :)

Reply:
Thank you for pointing it out! I've made the necessary changes!